src/Cms/PluginAssets.php

```diff
@@ -51,22 +51,25 @@ public static function clean(string $pluginName): void
 	 */
 	public static function resolve(
 		string $pluginName,
-		string $filename
+		string $path
 	): Response|null {
 		if ($plugin = App::instance()->plugin($pluginName)) {
-			$source = $plugin->root() . '/assets/' . $filename;
+			$assets = $plugin->assets();
 
-			if (F::exists($source, $plugin->root()) === true) {
+			if (
+				($asset = $assets[$path] ?? null) &&
```

Comment:
Could we use `$asset = $plugin->asset($path)` instead? Then we don't need the `$assets` variable declaration above.